tensorflow/core/kernels/mkl_fused_ops_test.cc

```diff
@@ -705,65 +743,25 @@ class FusedPadConvOpTest : public OpsTestBase {
     const Tensor& first = *GetOutput(0);
     const Tensor& second = *GetOutput(2);
     CommonTestUtilities<T> test_util;
-    test_util.ConvertAndCompare(dtype, first, second, expected);
+    test_util.ConvertAndCompareIntegral(dtype, first, second, expected);
   }
 };
 
-TEST_F(FusedPadConvOpTest, PaddingConvTest) {
-  const int depth = 1;
-  const int image_width = 4;
-  const int image_height = 3;
-  const int image_batch_count = 1;
-  Tensor image(DT_FLOAT, {image_batch_count, image_height, image_width, depth});
-  test::FillValues<float>(&image, {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12});
+TYPED_TEST_CASE_P(FusedPadConvOpTest);
```

Comment:
This is deprecated. Please use `TYPED_TEST_SUITE_P` instead.
```suggestion
TYPED_TEST_SUITE_P(FusedPadConvOpTest);
```